docs/docs/concepts/metrics-layer.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "What is a Metrics View?"
+sidebar_label: "What is a Metrics View?"
+sidebar_position: 11
+hide_table_of_contents: true
+---
+
+What is a metrics layer, and why did we decide to split the dashboard into two individual components? 
```

Comment:
Nit: If I have never used Rill before I don't care why we have done the split. The title and section aims to explain concepts in Rill that's current so it feels a bit odd that the main focus is on why we have done the split instead of explaning what a metrics view is.

Reply:
Makes sense! I would say to merge for now and come back to revise this specific page? (I believe Janet is also going to make proper visuals for this page)